_data/navigation.yml

```diff
@@ -1,43 +1,61 @@
 primary:
-  - text: Welcome
+  - text: About this guide
     href: /
   - text: Discover the current state
-    links:
-      - text: Overview
-        href: /discover/
-      - text: Understand users and their 'as is' process
-        href: /discover/users/
-      - text: Understand your stakeholders
-        href: /discover/stakeholders/
-      - text: Understand the technical landscape
-        href: /discover/technical/
-      - text: Understand the regulatory landscape
-        href: /discover/compliance/
+    href: /discover/
   - text: Define the future state
-    links:
-      - text: Overview
-        href: /future/
+    href: /define/
+  - text: Deliver the "right" way
```

Comment:
You might want to replace these with smart quotes, both for aesthetic and programmatic reasons. `option + [` for opening and `option + shift + [` for close should do it for you.

Reply:
👍